api_spec.yaml

```diff
@@ -153,57 +201,50 @@ paths:
         
         '500':
           $ref: '#/components/responses/500'
+              
 
-
-  /recipe/totp/device/list:
-    get:
+    delete:
       tags:
-          - Totp Recipe
-      operationId: getTotpDevices
+        - OAuth2 Recipe
+      operationId: revokeOAuth2AuthorizationCode
       description: |
-        Retrieve a list of TOTP devices for a user.
+        Revoke/Delete all authorization codes belonging to a session or to a client (across all tenants).
+        [Click here](https://app.code2flow.com/gPxFBmp7sdCR) for flow diagram.
+        To provide Single Logout, you should also call the normal session logout endpoint and the token revocation endpoint as well
       parameters:
-        - $ref: '#/components/parameters/totpRid'
+        - $ref: '#/components/parameters/oauth2Rid'
         - $ref: '#/components/parameters/api-key'
```

Comment:
need to also add `useDynamicSigningKey` (non optional) boolean to token POST API